src/sql/workbench/contrib/notebook/browser/notebookLinkHandler.ts

```diff
@@ -118,6 +118,25 @@ export class NotebookLinkHandler {
 		}
 	}
 
+	public getEncodedLinkUrl(): string | undefined {
+		// Need to encode URI here in order for user to click the proper encoded link in WYSIWYG
+		// skip encoding it if it's already encoded
+		if (typeof this._link === 'string') {
+			if (!this.containsEncodedComponents(this._link)) {
+				return encodeURI(this._link);
+			}
+			return this._link;
+		}
+		return undefined;
```

Comment:
I'd suggest adding a comment here about why you're returning undefined (I assume it's because there's no need to implement this currently?)

Reply:
yes, we're only handling strings from the link call out dialog.

Reply:
Please add that as a comment in the code